### ISSUE TITLE
Remove obsolete language version node v0.10.33 in favor of node v6.0.0 as default

### ISSUE THREAD

**Issue by HLFH** [Codewars/codewars.com#344]
Hi,

I'm getting some issues with [this method](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/String/repeat) on the [great kata](http://www.codewars.com/kata/570bbf7b6731d44b36001fde/train/javascript) of @myjinxin2015. 

![capture d ecran 2016-06-02 a 12 18 43](https://cloud.githubusercontent.com/assets/2359059/15741926/398f739c-28bc-11e6-8a58-281937683db8.png)

Could you solve it? And set node v6.0.0/Babel as default for Shortest Code : Jumping Dutch act? Because the kata creator can't modify it to "node v6.0.0".

Thanks in advance,
HLFH

**Comment by HLFH** [comment]
@xcthulhu @jhoffner Any ideas?

**Comment by HLFH** [comment]
@xcthulhu @jhoffner @slothpie @GiacomoSorbi

Hi,
 
The language version is Node 0.10.33 on [this other kata](https://www.codewars.com/kata/55a14aa4817efe41c20000bc/) but should be Node v6.
According to [node.green](http://node.green/), `class ChildClass extends ParentClass { ... }` are not supported with Node 0.10.33. Node v6 is recommended.
Could you change it?
It's the second time I have this issue... And I believe it's a deep issue than only the Codewars devs could change.

**Comment by HLFH** [comment]
Any ideas? @xcthulhu @jhoffner @slothpie @GiacomoSorbi

**Comment by jhoffner** [comment]
@HLFH Not all kata are available in all versions. The system checks all versions against the author's solution and code. In this case, there is something that is not compatible with Node 6. 

There was another kata that I saw you commented on (multiple times) which was actually broken and no version worked. I was able to look into that one and update the code to resolve the issue and it now runs in Node 6. I however don't have the bandwidth to try to update all kata that are not compatible in all versions. 

My suggestion would be that if a kata is solvable in at least one version, to try to solve it within that version. For what its worth, with the example kata that you gave, there is absolutely no reason why you would need to lean on ES6 features to solve it. Other than arrow functions (for syntactic sugar), not a single solution needed to use ES6.
